src/compile/mark/mark.ts

```diff
@@ -340,12 +367,154 @@ function getMarkGroup(model: UnitModel, opt: {fromPrefix: string} = {fromPrefix:
   ];
 }
 
+export function getLabel(model: UnitModel, data: string): LabelMark[] {
+  if (!model.encoding.label) {
+    return [] as LabelMark[];
```

Comment:
Why do we need `as LabelMark[]` here?

Reply:
I add this before I annotate the function return type. I will fix it.